ui/src/main/webapp/home/index.js

```diff
@@ -18,61 +15,12 @@ import Divider from 'material-ui/Divider'
 import { cyan500 } from 'material-ui/styles/colors'
 import RaisedButton from 'material-ui/RaisedButton'
 import MapDisplay from 'components/MapDisplay'
+import Spinner from 'components/Spinner'
 
+import * as selectors from './reducer'
```

Comment:
We should agree on some sort of code style requirements for imports. Personally, i prefer `* as` when all or most of the exports are being imported and they're very tightly related, or when names might conflict. It also makes it clearer in code which calls are actions, styles, and reducers without having to scroll to the top and skim through each potentially large import list. Name conflicts can also become an issue, and large refactoring can become tedious when you aren't sure which calls are being imported and which are local to that file.